app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt

```diff
@@ -86,7 +86,7 @@ fun CommentNodeHeader(
     score: Int,
     myVote: Int?,
     isModerator: Boolean,
-    onLongClick: () -> Unit = {}
+    onClick: () -> Unit = {}
```

Comment:
Follow the convention on these components, which is to make sure there's no default. IE 

`onClick: () -> Unit`